StaticAnalyzer/views/android/playstore.py

```diff
@@ -13,6 +13,9 @@ def get_app_details(package_id):
         det = app(package_id)
         det.pop('descriptionHTML', None)
         det.pop('comments', None)
+        description = det['description'].replace('<b>', '')
```

Comment:
@superpoussin22 Sorry for confusing., I meant `</br>` with `\n` 
`<b>` and `</b>` with ``

Reply:
Can be shortened

   det['description'] = det['description'].replace('<b>', '').replace('</b>', '').replace('<br>', '\n').replace('</br>', '\n')

Reply:
will do